### ISSUE TITLE
JAWS is not announcing for more button instructions or role: A11y_Visual Studio Code Client_Welcome Screen_Screen Reader.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/vscode#224154]
### GitHub Tags:
#A11yWCAG2.1; #A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM_Visual Studio Code Client_Win32_JULY2024; #DesktopApp; #WCAG1.3.1; #FTP; #A11yMAS; #info and relationships; #Element:button; #JAWS; #NVDA; #Screen reader; #A11yMAS; #STP;

### Environment and OS details:
Application Name: Visual Studio Code Client
OS: Windows 11 version 23H2 OS built: 22631.3880.
Screen reader: NVDA: version 2024.1, JAWS 2024. 2312.53

### Reproduction Steps:

1.Turning on the JAWS/ NVDA.
2.Open Visual studio code insider editor.
3.Using "Alt+H" key navigate to the "Help" menu bar option.
4.Using "Down" arrow key navigate to the "Welcome" item.
5.Tab till more button.
6.observed that JAWS is not announcing for more button instructions or role

### Actual Result:
JAWS is not announcing for more button instructions or role.
**Note:**
Similar issue not observed with NVDA.

### Expected Result:
JAWS should announce for more button instructions or role

### User Impact:
Screen reader users will face difficulty. There are not able know more is button or link or text. if Jaws is not announcing anything.

### Attachments:

**Comment by Samhitha66** [comment]
#Rev:samhithaks;
